qiskit/providers/ibmq/api_v2/clients/auth.py

```diff
@@ -13,18 +13,23 @@
 # that they have been altered from the originals.
 
 """Client for accessing authentication features of IBM Q Experience."""
+from typing import Dict, List
 
 from ..exceptions import AuthenticationLicenseError, RequestsApiError
 from ..rest import Api, Auth
 from ..session import RetrySession
 
 from .base import BaseClient
 
+# Type aliases for type hints.
```

Comment:
Some annotations were used multiples times, so I defined type aliases.

Reply:
Can you actually remove the aliases for the time being? The mean reason is starting the effort "simple and explicit", and we can revise using aliases once we are a bit more advanced in the typing effort if needed.